src/main/scala/mastermind/Mastermind.scala

```diff
@@ -13,19 +13,26 @@ val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
 def getRandomColor(): Color =
-  ???
+  val random = new Random
+  validColors(random.nextInt(validColors.length)).toChar
 
 /** Given four colors, make a board from them */
 def makeBoardFromColors(c1: Color, c2: Color, c3: Color, c4: Color): Board =
-  ???
+  c1.toString() + c2.toString() + c3.toString() + c4.toString()
```

Comment:
I did this differently. Instead of converting each char to a string, I just added them to an empty string and it converted them automatically.